helix-core/src/main/java/org/apache/helix/monitoring/metrics/implementation/RebalanceLatencyGauge.java

```diff
@@ -50,24 +54,24 @@ public RebalanceLatencyGauge(String metricName, long slidingTimeWindow) {
   @Override
   public void startMeasuringLatency() {
     reset();
-    _startTime = System.currentTimeMillis();
+    _startTime.set(System.currentTimeMillis());
   }
 
   /**
    * WARNING: this method is not thread-safe.
```

Comment:
nit: update the comment? After your change, the class should be thread-safe implementation, right?

Reply:
Good catch. Will fix.